quickstart.md

```diff
@@ -434,8 +434,162 @@ root@localhost:/# ls /run/image/
 layers  lost+found  overlay
 ```
 
-# Debugging problems
+# Troubleshooting
+
+Confidential Containers integrates several components. If you run into problems,
+it can sometimes be difficult to figure out what is going on or how to move forward.
+Here are some tips.
+
+## Kubernetes
+
+To figure out which basic area you problem is in, first make sure that your Kubernetes
+cluster can schedule non-confidential workloads on your worker node. Remove the `kata-*`
+runtime class from your pod yaml and try to run a pod. If your pod still doesn't run,
+please refer to a more general Kubernetes troubleshooting guide.
+
+If your cluster is healthy but you cannot start confidential containers, you might
+be able get some helpful information from Kubernetes.
+Try `kubectl describe pod <your-pod>`
+Sometimes this will give you a useful message pointing to a faied attestation
+or some sort of missing environment setup. Most of the time you will see a
+generic message such as the following:
+
+```
+Failed to create pod sandbox: rpc error: code = Unknown desc = failed to create containerd task: failed to create shim: Failed to Check if grpc server is working: rpc error: code = DeadlineExceeded desc = timed out connecting to vsock 637456061:1024: unknown
+```
+
+Unfortunately this is a generic message. You'll need to go deeper to figure out
+what is going on.
+
+## CoCo Debugging
+
+A good next step is to figure out if things are breaking before or after the VM boots.
+You can see if there is a hypervisor process running with something like this.
+```bash
+ps -ef | grep qemu
+```
+
+If you are using a different hypervisor, adjust command accordingly.
+If there are no hypervisor processes running on the worker node, the VM has
+either failed to start or was shutdown. If there is a hypervisor process,
+the problem is probably inside the guest.
+
+Now is a good time to enable debug output for Kata and containerd.
+To do this, first look at the containerd config file located at
+`/etc/containerd/config.toml`. At the bottom of the file there should
+be a section for each runtime class. For example:
+
+```
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-qemu-sev]
+  cri_handler = "cc"
+  runtime_type = "io.containerd.kata-qemu-sev.v2"
+  privileged_without_host_devices = true
+  pod_annotations = ["io.katacontainers.*"]
+  [plugins."io.containerd.grpc.v1.cri".containerd.runtimes.kata-qemu-sev.options]
+    ConfigPath = "/opt/confidential-containers/share/defaults/kata-containers/configuration-qemu-sev.toml"
+```
+
+The `ConfigPath` entry on the final line shows the path to the Kata configuration file that will be used
+for that runtime class.
+
+While you are looking at the containerd config, find the `[debug]` section near the top and  set `level`
+to `info`. Make sure to restart containerd after editing the containerd config file.
```

Comment:
should the `level` be `debug` ? I think default is `info` in containerd .

Reply:
good thing someone is paying attention